Fabric.Authorization.API/scripts/AzureAD-Users-Migration.ps1

```diff
@@ -376,19 +374,20 @@ function Get-ADUsers
     {
        # cleanup variables 
        # if AD User is mispelled the values from the previous user will be entered for the next user
-       $pc = $null
        $samAccountName = $null
        $entry = $null
        $userMap = $null
        $userMapGuid = $null
 
        [string] $subjectId = $user.SubjectId.ToString()
-       # find the samAccountName in AD from the AuthorizationDB SubjectId
-       $pc = Get-PrincipalContext -domain $domain
-       $samAccountName = Get-SamAccountFromAccountName -accountName $subjectId
-
+       # This script is in Install-Authorization-Utilities.psm1
+       $samAccountName = Get-SamAccountFromAccountName -accountName $subjectId 
+       $samDomainName = Get-SamDomainFromAccountName -accountName $subjectId
+       
        # get all properties of the user with $samAccountName
-       $entry = Get-ADUser $samAccountName -Properties *
+       # You can find this script in ActiveDirectory module, located here:
+       # Remote Server Administration Tools https://www.microsoft.com/en-us/download/details.aspx?id=45520
+       $entry = Get-ADUser $samAccountName -Server $samDomainName -Properties * 
```

Comment:
hopefully nothing too crazy happens when try to search a domain it can't talk to